### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by Aqilzeka**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Aqilzeka** [comment]
Add Docker list two resource

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

Pleas close the other PR

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -382,6 +382,8 @@
 * [Docker](https://www.youtube.com/playlist?list=PLhW3qG5bs-L99pQsZ74f-LC-tOEsBp2rK) - Raghav Pal
 * [Docker Tutorial for Beginners ](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGBkvpSIgwchk0glHLz7CQ-7) - Thenewboston
 * [Docker Tutorial for Beginners](https://www.youtube.com/watch?v=fqMOX6JJhGo) - FreeCodeCamp
+* [Docker Tutorial for Beginners [FULL COURSE in 3 Hours]](https://www.youtube.com/watch?v=3c-iBn73dDE&t=2721s) - TechWorld with Nana
+* [Docker Tutorial for Beginners | Full Course [2021]](https://www.youtube.com/watch?v=p28piYY_wv8) - Amigoscode
```
Comment (davorpa) [inline courses/free-courses-en.md:386]: Remove video timing param from ulrs and escape pipes from titles

```suggestion
* [Docker Tutorial for Beginners [FULL COURSE in 3 Hours]](https://www.youtube.com/watch?v=3c-iBn73dDE) - TechWorld with Nana
* [Docker Tutorial for Beginners \| Full Course [2021]](https://www.youtube.com/watch?v=p28piYY_wv8) - Amigoscode
```

**Review by davorpa — CHANGES_REQUESTED** [review]

**Comment by davorpa** [comment]
> For youtube, we prefer "playlist" links and the name of the producer/creator. [https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md?rgh-link-date=2022-06-08T12%3A15%3A32Z#platforms-and-access-notes)

Both are 3h lvideos with chapters integrated in timeline, so it's ok for me :wink: 

> Pleas close the other PR

PR #6862 closed as duplicated

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -382,6 +382,8 @@
 * [Docker](https://www.youtube.com/playlist?list=PLhW3qG5bs-L99pQsZ74f-LC-tOEsBp2rK) - Raghav Pal
 * [Docker Tutorial for Beginners ](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGBkvpSIgwchk0glHLz7CQ-7) - Thenewboston
 * [Docker Tutorial for Beginners](https://www.youtube.com/watch?v=fqMOX6JJhGo) - FreeCodeCamp
+* [Docker Tutorial for Beginners [FULL COURSE in 3 Hours]](https://www.youtube.com/watch?v=3c-iBn73dDE) - TechWorld with Nana
+* [Docker Tutorial for Beginners \| Full Course [2021]](https://www.youtube.com/watch?v=p28piYY_wv8) - Amigoscode
```
Comment (davorpa) [inline courses/free-courses-en.md:386]: Escape markdown tokens

```suggestion
* [Docker Tutorial for Beginners \[FULL COURSE in 3 Hours\]](https://www.youtube.com/watch?v=3c-iBn73dDE) - TechWorld with Nana
* [Docker Tutorial for Beginners \| Full Course \[2021\]](https://www.youtube.com/watch?v=p28piYY_wv8) - Amigoscode
```

**Review by davorpa — APPROVED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md]: Resolved by commit 8908450b2aadef2af88b881f2ed5d3578182e8c4

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**